### ISSUE TITLE
Tabindex Paella Player - Accessibility

### ISSUE THREAD

**Issue by NadiaUoM** [polimediaupv/paella-player#61]
As part of our accessibility testing process, we utilize the WAVE Evaluation Tool to assess and improve web accessibility. During this evaluation, we identified the concept of Tab Index, which is explained in detail in the following section:

**What It Means**
A positive tabindex value is present.

**Why It Matters**
Tabindex values of 1 or greater specify an explicit tab/navigation order for page elements. Because it modifies the default tab order, cause confusion, and result in decreased keyboard accessibility, it should be avoided.

**How to Fix It**
If the natural tab order is already logical, remove the tabindex. Otherwise, consider restructuring the page so that tabindex is not needed. If tabindex is maintained, ensure that the resulting navigation is logical and complete.
A positive tabindex value is present.

<img width="699" height="94" alt="Image" src="https://github.com/user-attachments/assets/ca035470-24a3-486d-9a00-41a70bbce7fb" />
<img width="673" height="67" alt="Image" src="https://github.com/user-attachments/assets/00b5c909-9a14-4009-a766-4793d50f24cf" />

The problem
There is actually bug in this file.

1. Positive tabindex values are being generated
This function:

export function getNextTabIndex(player) {
    player.__tabIndex = player.__tabIndex || 0;
    ++player.__tabIndex;
    return player.__tabIndex;
}
creates sequential values:

1
2
3
4
This code is located in ButtonPlugin.js

Those values are then used here:

const tabIndex = plugin.tabIndex;
and rendered into the HTML:

<button
    ...
    tabindex="${tabIndex}"
    ...
>
So your buttons become:

<button tabindex="41"> <button tabindex="42">

This is exactly what accessibility checkers warn about.

**Comment by ferserc1** [comment]
I don’t understand what the problem is. Could you please explain it in more detail?

If you’re referring to the fact that the buttons in the video layout are explicitly assigned a higher tab index, and as a result these buttons are activated later when you press the Tab key, **this is entirely intentional**. You cannot assume that just because a button appears earlier in the interface, it must be activated first. In fact, the system is designed so that the first elements activated using the Tab key are the most important ones: play, seek, volume, etc.

However, this setting is configurable. The initial tab index, from which the rest of the video layout’s indices are calculated, is configured within the plugin itself:

```json
"es.upv.paella.dualVideoDynamic": {
            "enabled": true,
            "validContent": [
                { "id": "presenter-presentation-dynamic", "content": ["presentation","presenter"], "icon": "present-mode-3.svg", "title": "Presenter and presentation" },
                { "id": "presenter-2-presentation-dynamic", "content": ["presenter-2","presentation"], "icon": "present-mode-3.svg", "title": "Presenter and presentation" },
                { "id": "presenter-presenter-2-dynamic", "content": ["presenter","presenter-2"], "icon": "present-mode-3.svg", "title": "Presenter and presentation" }
            ],
            "pipContentIds": [
                "presenter-presentation-pip",
                "presenter-2-presentation-pip",
                "presenter-presentation-2-pip"
            ],
            "allowSwitchSide": false,
            "tabIndexStart": 40
        },
```

Therefore:
1. The player has been configured in this way specifically for accessibility.
2. If you do not wish to implement this behaviour by default, both the initial tab index of the video layouts and the tab index of the individual buttons can also be configured.
